crates/orchestrator/src/network/node.rs

```diff
@@ -45,6 +46,14 @@ impl NetworkNode {
         Ok(())
     }
 
+    pub async fn rpc(&self) -> Result<RpcClient, subxt::Error> {
+        RpcClient::from_url(&self.ws_uri).await
+    }
+
+    pub async fn client(&self) -> Result<OnlineClient<PolkadotConfig>, subxt::Error> {
```

Comment:
Does make sense to allow users to also specify the concrete `Config` type?

Reply:
mmm good point. Using moonbeam for example you would need to specify a custom config.
A bit of a shame that Rust does not let you specify a default value for the generic trait here.

I mean you could but then rustc complains with
```
defaults for type parameters are only allowed in `struct`, `enum`, `type`, or `trait` definitions  this was previously accepted by the compiler but is being phased out;
```